src/components/HTMLEngineProvider/HTMLRenderers/AnchorRenderer.js

```diff
@@ -47,6 +48,11 @@ const AnchorRenderer = (props) => {
         // If we are handling a New Expensify link then we will assume this should be opened by the app internally. This ensures that the links are opened internally via react-navigation
         // instead of in a new tab or with a page refresh (which is the default behavior of an anchor tag)
         if (internalNewExpensifyPath) {
+            if (attrPath.indexOf('r/') === 0) {
```

Comment:
NAB: should we check for this instead?
```suggestion
            if (attrPath.startsWith('r/') === 0) {
```